ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -101,6 +101,24 @@ tests:
     - ref: ipi-install-heterogeneous
```

Comment:
So this means both amd and arm are covered, right?

Reply:
right, but only for workers.